internal/pass/sign/verify.go

```diff
@@ -0,0 +1,43 @@
+package sign
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var ErrInvalidClaims = errors.New("invalid claims")
+
+// CanI establish connection with type tp given claims in token.
+func (s Service) CanI(tokenString string, ct ConnectionType) error {
```

Comment:
we want connection_id to be also returned from verify method. That way we don;t need to pass extension_id in each request.

Reply:
Let's do it once we start "merging" our changes. We will then know, what is actually needed. I think `sign` package is POC right now.